prepareComments.js

```diff
@@ -0,0 +1,61 @@
+
+fs = require('fs');
+const data = fs.readFileSync('eslint.json', 'utf8');
+const myArgs = process.argv.slice(2);
+commitId = myArgs[0]
+
+function main(){
+    prepareComment(data);
+}
+
+function createComments(comPath, comCommitId, comLine, comBody, fileName) {
+    console.log("IN FUNCTION -->>"  + comLine );
+    let com = {
+        path: comPath,
+        commit_id: comCommitId,
+        line: comLine,
+        side: "RIGHT",
+        body: comBody
+    };
+    fs.writeFile('eslintComments/file' + fileName + 'jslint.json', JSON.stringify(com), (err) => {
+        if (err) {
+            throw err;
+        } else {
+            console.log('The file has been saved!');
+        }
+    });
+}
+
+function prepareComment(content) {
+    var parseContent = JSON.parse(content);
+    var mapLines = new Map();
+    parseContent.forEach(function (file, fisrtIndex) {
+        var comPath = file.filePath.split('\\changedFiles\\')[1];
+        file.messages.forEach(function(message, secondIndex) {
+            let keyMapProp = {
```

Comment:
Expected indentation of 6 spaces but found 12.
'keyMapProp' is never reassigned. Use 'const' instead.